_pkgdown.yml

```diff
@@ -14,50 +19,26 @@ template:
       creator: "@rstudio"
       card: summary
 
-development:
-  version_tooltip: "Development version"
-
 home:
-  strip_header: false
   links:
   - text: Learn more
     # if relevant, set url to another website i.e https://rmarkdown.rstudio.com
     href: ""
 
 # structure of website themed with quillt
 navbar:
-  title: ~
-  type: default
   structure:
-    left:  [intro, examples, articles]
-    right: [reference, news, github]
+    left:  [intro, examples, articles, reference, news]
   components:
-    home: ~
     examples:
       text: Examples
       href: articles/articles/examples.html
-    articles:
-      text: Articles
-      menu:
-        - text: Testing quillt style
-          href: articles/articles/test-style.html
```

Comment:
After testing, I remember now that I was specifiying explicitly because setting a specific category for example was not preventing the example article to be put also in articles menu. Removing that will make the article shown twice in the navbar unfortunatly. Do you think there is a way to prevent that ? 
Otherwise, I don't mind specifying manually which is what we are doing now.

Reply:
Yes, you can now control how articles appear in the navbar at https://pkgdown.r-lib.org/dev/reference/build_articles.html#index-and-navbar. Otherwise you'd have to remember to keep the articles menu up-to-date every time you added or renamed a vignette.

Reply:
Great I'll use that then.